common.h

```diff
@@ -23,3 +23,7 @@ enum Channel {
 	BLUE = 2,
 	ALPHA = 3
 };
+
+template <typename To, typename From> To cast(From value) {
+	return static_cast<To>(value);

```

Comment:
Pure convenience for writing a static_cast with less typing because we have to do it somewhat frequently. This probably wasn't necessary and might actually be slightly harmful in the long term for reasons I am too tired to explain but whatever.